swagger_to_sdk_config_track2.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/documentation/sdkautomation/SwaggerToSdkConfigSchema.json",
+  "generateOptions": {
+    "generateScript": {
+      "path": "generator automation-track2",
```

Comment:
We have been trying to avoid the name codename "track2" from many places in our repo and instead we have been trying to use the version of the code generator. Can we please try to come up with a naming convention that doesn't include "track2" in all the names of files and configurations?

Reply:
How about `new-version` like the branch name in `azure-sdk-for-go-samples`?

Reply:
According to @ArcturusZhang , I've changed the name in script. Cause the files have been moved to eng folder, the filename is no need to user track2 suffix now.

Reply:
While "new-version" is better then "track2" it is still a little shortsighted because what happens when we have the next new version? If we have a version number for the generator that would be preferred and that is what other languages are doing. If there isn't an explicit version perhaps we just call this v2 so that we have a version for it.

Also there looks to still be a few references to track2 in files, like the directory. Lets consider changing those to v2 instead.

Reply:
v2 is a resonable name. I've changed all the track2 to v2.